sig-scalability/extending_slo.md

```diff
@@ -0,0 +1,71 @@
+# Extended Kubernetes scalability SLOs
```

Comment:
We'll need to figure out how to stage these out release by release. Thinking we'll cover that in the feature repo, though. This is looking good.